### DISCUSSION TITLE
Connecting Røde VideoMic NTG possible?

### DISCUSSION THREAD

**Discussion by kongjudas** [mcguirepr89/BirdNET-Pi#1229] [Q&A] [answered]
I'm struggling to find some way to connect my Røde VideoMic NTG to the Pi installation (Pi 4). It's a USB mic and I don't see it anywhere, not even a setting where I can include it. Please give me a short write-up on how to get it working...

Edit: My bad! I had a charging only USB cable. **Please delete this**. I don't know how.

**Comment by PaulRB** [comment, marked as answer]
Look for this button and click it: 
![Screenshot_20240807-135242-945](https://github.com/user-attachments/assets/937f44a6-d71c-4a22-b8e5-b3adf86c628a)

So if your mic is now working, what is your opinion of this model for use with BirdNetPi? How much noise or hum? How loud and clear are the bird songs?

**Reply by kongjudas** [reply]
The mic is awesome. I have no scientific measurements of noise, but I don't really hear any. I use it with the built-in 150 hz low pass filter, so it removes some of the traffic and sorrounding noise. I also have the gain turned up to 7 and use it with a deadcat. The sound quality is really good, depending on the distance and interfering leaf and wind in the trees..

Compared to some of the camera mic's (Tapo with RTSP)  it's a different world. I do NOT recommend the Tapo cameras. A bit off topic, but here's the Tapo for those who are interested:

<img width="692" alt="Skjermbilde 2024-08-15 kl  12 33 16" src="https://github.com/user-attachments/assets/767ecbfc-d735-46b0-ac67-27e9f199fe93">

**Comment by Bongo2** [comment]
@kongjudas 
I also connected this microphone via USB. But it doesn't work.
What do I have to enter in the audio settings? Or do you have to configure something else in the operating system?

Info would be very kind.

**Reply by PaulRB** [reply]
Could you have you made same mistake as @kongjudas, your usb cable is for charging only, not data?

**Comment by Bongo2** [comment]
I thought I had the right USB cable. I exchanged it and now it works.
I don't have to configure anything. Only plug in.

Thank you.
